CFEScraping/index.js

```diff
@@ -0,0 +1,13 @@
+const puppeteer = require('puppeteer');
```

Comment:
All this logic should be in an independent file and called to be used as an external library here

Reply:
Name the folder with lowercase & snake case if needed